yellowbrick/classifier/prcurve.py

```diff
@@ -86,6 +86,9 @@ class PrecisionRecallCurve(ClassificationScoreVisualizer):
         Draw ISO F1-Curves on the plot to show how close the precision-recall
         curves are to different F1 scores.
 
+    f1_values : list , default=[0.2,0.4,0.6,0.8]
```

Comment:
Great add! For consistency might I recommend that this arg be named `iso_f1_values`?

Reply:
sure. duly noted.